src/hooks/useGetPictureOfTheDay.js

```diff
@@ -0,0 +1,16 @@
+import useAxios from "@hooks/useAxios";
+
+function useGetPictureOfTheDay(date) {
+  const { response, loading } = useAxios({
+    method: 'get',
+    params: {
+      api_key: 'xMFt22A9Jl7xcwVUYFY7wNqne4K6kJaIEpaiU0T9',
```

Comment:
It's okay for this project JFYI https://create-react-app.dev/docs/adding-custom-environment-variables/